src/pagination/pagination.js

```diff
@@ -97,6 +101,23 @@ export default class Pagination extends React.PureComponent<
     this.setState({isMenuOpen});
   };
 
+  constructAriaWayfinderLabel = (locale: object, prefix: string) => {
```

Comment:
@chasestarr 
I know I'm not doing this type checking correctly. Any suggestions?

Reply:
`object` can be replaced with `LocaleT` from [here](https://github.com/uber-web/baseui/blob/26a8b9a38ef5621d5055fd185ca3a88b5a8804ca/src/locale/types.js)